libloki/api.js

```diff
@@ -196,9 +168,8 @@
   window.libloki.api = {
     sendSessionEstablishedMessage,
     sendSessionRequestsToMembers,
-    createContactSyncProtoMessage,
-    createGroupSyncProtoMessage,
-    createOpenGroupsSyncProtoMessage,
+    createContactSyncMessage,
```

Comment:
Best to not put stuff in libloki as we're going to remove it in the future anyway. Maybe a file in obsession would be better

Reply:
we will move this out after the refactor is done